RadixWallet/Clients/KeychainClient/KeychainClient+Interface.swift

```diff
@@ -16,6 +16,7 @@ public struct KeychainClient: Sendable {
 	public var _removeDataForKey: RemoveDataForKey
 	public var _removeAllItems: RemoveAllItems
 	public var _getAllKeysMatchingAttributes: GetAllKeysMatchingAttributes
+	public var _keychainChanged: KeychainChanged
```

Comment:
Maybe we ought write a note that this is a "best effort" publisher, it does not detect external changes, such as Keychain getting wiped when a passcode is disabled. So that we neither nor, nor any colleague in future assumes it always emits for all changes.